src/components/Common/AuthForm/AuthForm.tsx

```diff
@@ -49,6 +49,7 @@ const AuthForm: FC<AuthFormProps> = ({ formType }) => {
   const toast = useToast();
 
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [showAlert, setShowAlert] = useState<boolean>(false);
```

Comment:
Can we use useDisclosure hook here? so we don't have to maintain the state ourself.

Reply:
Done